beacon-chain/db/iface/interface.go

```diff
@@ -102,7 +103,7 @@ type HeadAccessDatabase interface {
 	EnsureEmbeddedGenesis(ctx context.Context) error
 
 	// initialization method needed for origin checkpoint sync
-	SaveOrigin(ctx context.Context, state io.Reader, block io.Reader) error
+	SaveOrigin(ctx context.Context, state []byte, block []byte) error
```

Comment:
I could see myself misusing it by passing in a state root and block root. Maybe we can make the arg names more specific

Reply:
renamed these `serState` and `serBlock`, lmk if you have a suggestion for something more clear.